src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -160,7 +160,8 @@ public void actionPerformed(final ActionEvent e) {
             + "<br><b> Other Things</b><br>"
             + "Press 'N' to cycle through units with movement left (move phases only).<br>"
             + "Press 'F' to highlight all units you own that have movement left (move phases only).<br>"
-            + "Press 'I' or 'V' to popup info on whatever territory and unit your mouse is currently over.<br>";
+            + "Press 'I' or 'V' to popup info on whatever territory and unit your mouse is currently over.<br>"
+            + "Press 'u' while mousing over a unit to undo all moves that unit has made (beta).<br>";
```

Comment:
#347 opened to lower case the others. The feature is 'u', not 'shift+u', same goes for the other keys.